deps/simdutf/simdutf.gyp

```diff
@@ -0,0 +1,148 @@
+{
+  'variables': {
+    'arm_fpu%': '',
+    'target_arch%': '',
+  },
+
+  'targets': [
+    {
+      'target_name': 'simdutf',
+      'type': 'static_library',
+      'include_dirs': ['.'],
+      'sources': ['simdutf.cpp'],
```

Comment:
Compiling the same exact source file merely with different cflags isn't what I had in mind.

After a brief look at the simdutf code, it looks like it does compile time detection for ARM and PPC and runtime detection for x86.

Also be aware that the library says it is not optimized on 32-bit and that performance may be poor on such CPUs.

Reply:
I updated the gyp file, but I'm having some trouble with AVX512. Do you have any recommendations? Regarding 32 bits, we can fallback to v8, if possible.